modules/web/sites/.test/functionAppCommon/main.test.bicep

```diff
@@ -176,5 +178,11 @@ module testDeployment '../../main.bicep' = {
     userAssignedIdentities: {
       '${nestedDependencies.outputs.managedIdentityResourceId}': {}
     }
+    hybridConnectionRelays: [
+      {
+        resourceId: nestedDependencies.outputs.hybridConnectionResourceId
```

Comment:
This should go away once the readme is generated

Reply:
I'm not following why this hybridConnectionRelays block needs to go away?

Reply:
All good. The resource ID field. Refreshing the readme should close this comment